src/coreclr/jit/lsraarm64.cpp

```diff
@@ -1430,6 +1431,19 @@ int LinearScan::BuildHWIntrinsic(GenTreeHWIntrinsic* intrinsicTree, int* pDstCou
                         assert(intrin.op4->isContainedIntOrIImmed());
                         break;
 
+                    case NI_Sve_CreateTrueMaskByte:
+                    case NI_Sve_CreateTrueMaskDouble:
+                    case NI_Sve_CreateTrueMaskInt16:
+                    case NI_Sve_CreateTrueMaskInt32:
+                    case NI_Sve_CreateTrueMaskInt64:
+                    case NI_Sve_CreateTrueMaskSByte:
+                    case NI_Sve_CreateTrueMaskSingle:
+                    case NI_Sve_CreateTrueMaskUInt16:
+                    case NI_Sve_CreateTrueMaskUInt32:
+                    case NI_Sve_CreateTrueMaskUInt64:
+                        needBranchTargetReg = !intrin.op1->isContainedIntOrIImmed();
```

Comment:
this creates internal register for "def". Make sure that we create an internal register for "use" as well. I forgot to do that in one place and fixed it in https://github.com/dotnet/runtime/pull/98814.